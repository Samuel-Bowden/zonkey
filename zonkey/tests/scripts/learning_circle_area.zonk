function circle_area(Float radius) -> Float {
    let pi = 3.14159;
    return pi * powerf(radius, 2.);
}

start {
    println(circle_area(5.0));
    println(circle_area(10.0));
    println(circle_area(2.5));
}
