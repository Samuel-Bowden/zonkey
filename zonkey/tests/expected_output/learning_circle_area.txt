78.53975
314.159
19.6349375
